privacy-security-explainer.md

```diff
@@ -36,13 +36,10 @@ It is often necessary to be sure of user intent before exposing sensitive inform
 A User Agent may use implied consent based, for example, on the install status of a web application or frequency and recency of visits. Given the sensitivity of XR data, caution is strongly advised when relying on implicit signals. 
 
 ### Explicit consent
-It is often useful to get explicit consent from the user before exposing sensitive information. When gathering explicit user consent, User Agents present an explanation of what is being requested and provide users the option to decline. Requests for user consent can be presented in many visual forms based on the features being protected and User Agent choice. While often associated with the [Permissions API](https://www.w3.org/TR/permissions/), the concept of user consent does not have exact overlap.
-
-### Timing of XR-related consent prompts
-**TODO** Fill this in with what was agreed up on in [#720](https://github.com/immersive-web/webxr/issues/720).
+It is often useful to get explicit consent from the user before exposing sensitive information. When gathering explicit user consent, User Agents present an explanation of what is being requested and provide users the option to decline. Requests for user consent can be presented in many visual forms based on the features being protected and User Agent choice. While often associated with the [Permissions API](https://www.w3.org/TR/permissions/), the concept of user consent does not have exact overlap. If sensitive data is protected by explicit consent and will be used during an `XRSession`, it is strongly recommended that User Agents display the associated consent prompt prior to the session being created.
 
 ### Duration of consent
-**TODO** Fill this in with what was agreed up on in [#721](https://github.com/immersive-web/webxr/issues/721).
+It is strongly recommended that once consent is granted, either implicitly or explicitly, for a specific [origin](https://html.spec.whatwg.org/multipage/origin.html) that this consent persist until the [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context) has ended. User agents may also choose to allow consent to persist for all subsequent browsing contexts of that origin. Caution is advised when doing do, especially when done in response to implied consent.
```

Comment:
My impression previously is that we wanted access to features consent was granted for to be scoped to session lifetime. (As you stated here, the consent itself could persist between sessions in the same browsing context.) Where would be the appropriate place to make that distinction? In this section, or is there another stub that's earmarked for it?

Reply:
I'm not sure what you're referring to?  This text was intended to reflect my understanding of the consensus reached in https://github.com/immersive-web/webxr/issues/721#issuecomment-504614488 and https://github.com/immersive-web/webxr/issues/721#issuecomment-505224211.  @johnpallett and @avadacatavra, did I misunderstand?

Reply:
You know what, it's not actually relevant at this point. Previous conversations were around things like "granting a session access to the camera shouldn't grant that access for the entire page" but that's not something that we're considering at this stage. The features that we ARE discussing (primarily reference space access) are naturally session-bound anyway, so it's a moot point. This is fine.